Packs/PaloAltoNetworks_IoT3rdParty/Integrations/PaloAltoNetworksIoT3rdParty/PaloAltoNetworksIoT3rdParty.yml

```diff
@@ -0,0 +1,172 @@
+        A Null value will return full inventory (1000 MAX)'
+      name: incrementTime
+    - description: '(Optional)Offset for paging: Null value will accumulate all results
+        by default.'
+      name: offset
+    - description: '(Optional)Page size for paging: Null value will accumulate all
+        results by default.'
+      name: pageLength
+    description: Returns a list of assets for the specified asset type.
+    name: panw-iot-3rd-party-get-asset-list
+    outputs:
+    - contextPath: PanwIot3rdParty.Devices
+      description: List of devices.
+    - contextPath: PanwIot3rdParty.Alerts
+      description: List of Alerts.
+    - contextPath: PanwIot3rdParty.Vulnerabilities
+      description: List of Vulnerabilities.
+  - arguments:
+    - auto: PREDEFINED
+      description: Type of Asset.
+      name: assetType
+      predefined:
+      - Device
+      - Alert
+      - Vulnerability
+      required: true
+    - description: Asset ID. MacAddress for device, zb_ticketid for alert and vulnerability.
+      name: assetID
+      required: true
+    description: For a given a asset ID (alert-id, vulnerability-id or mac-address)
+      returns the asset details.
+    name: panw-iot-3rd-party-get-single-asset
+    outputs:
+    - contextPath: PanwIot3rdParty.SingleAsset
+      description: Asset Details.
+  - arguments:
+    - auto: PREDEFINED
+      description: Status to be reported to PANW IoT Cloud.
+      name: status
+      predefined:
+      - error
+      - success
+      - disabled
+      required: true
+    - description: Message to be sent to PANW IoT Cloud.
+      name: message
+      required: true
+    - auto: PREDEFINED
+      description: Name of PANW IoT 3rd Party Integration.
+      name: integrationName
+      predefined:
+      - siem
+      - ise
+      - servicenow
+      required: true
+    - description: Name of the playbook.
+      name: playbookName
+      required: true
+    - auto: PREDEFINED
+      description: Type of asset associated with the status report.
+      name: type
+      predefined:
+      - device
+      - alert
+      - vulnerability
+      required: true
+    description: Sends a status message back to PANW IOT cloud.
+    name: panw-iot-3rd-party-report-status-to-panw
+  - arguments:
+    - auto: PREDEFINED
+      description: Input asset type.
+      name: assetType
+      predefined:
+      - Device
+      - Alert
+      - Vulnerability
+      required: true
+    - auto: PREDEFINED
+      description: Desired output format.
+      name: outputFormat
+      predefined:
+      - CiscoISECustomAttributes
+      - SIEM
+      - ServiceNow
+      required: true
+    - description: List of input assets.
+      isArray: true
+      name: assetList
+    - description: 'Example: ServiceNow ID and deviceid mapping.'
+      isArray: true
+      name: metadata
+    - description: 'Example: incident triggered by PANW IoT cloud API.'
+      name: metadata1
```

Comment:
can you explain?

Reply:
changed the names. 
"metadata" to "incident" -> We invoked the incident API and pass a lot fields (CustomFields) . This incident is meant to passed as input so we can use those custom fields. 
"metadata1" to "servicenow_map" -> these are serviceNow ID and deviceID mappings